package.json

```diff
@@ -34,30 +35,22 @@
   },
   "dependencies": {
     "@nextcloud/router": "^2.0.0",
-    "core-js": "^3.6.4",
     "dompurify": "^2.4.1",
     "escape-html": "^1.0.3",
     "node-gettext": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.19.3",
-    "@babel/core": "^7.20.5",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-typescript": "^7.18.6",
-    "@happy-dom/jest-environment": "^8.1.0",
     "@nextcloud/browserslist-config": "^2.3.0",
```

Comment:
can be removed too

Reply:
Browserslist config? This is still used by `check-es-compat`

Reply:
:+1: